renderer/main.js

```diff
@@ -549,7 +549,7 @@ function onOpen (files) {
 function isTorrent (file) {
   var name = typeof file === 'string' ? file : file.name
   var isTorrentFile = path.extname(name).toLowerCase() === '.torrent'
-  var isMagnet = typeof file === 'string' && /^magnet:/.test(file)
+  var isMagnet = typeof file === 'string' && /^magnet:/.test(file) || /^stream-magnet:/.test(file)
```

Comment:
This also could be like `var isMagnet = typeof file === 'string' && /^(stream-)?magnet:/.test(file)`